web/test/pages/profile/profile-starting.test.tsx

```diff
@@ -1329,6 +1329,98 @@ describe("profile - starting business", () => {
     });
   });
 
+  describe("profile opportunities alert for Poppy", () => {
+    const ProfileConfig = Config.profileDefaults.fields;
+
+    it("only displays alert on info tab", () => {
+      const business = generateBusinessForProfile({
+        profileData: generateProfileData({
+          operatingPhase: "UP_AND_RUNNING",
+          businessPersona: "STARTING",
+          dateOfFormation: undefined,
+        }),
+      });
+      renderPage({ business });
+      expect(screen.getByTestId("opp-alert")).toBeInTheDocument();
+      chooseTab("numbers");
+      expect(screen.queryByTestId("opp-alert")).not.toBeInTheDocument();
+    });
+
+    it("lists each unanswered funding/certification question", () => {
```

Comment:
[Dust] I understand after reading the test that funding/certification questions refer to `existing employees, homeBasedBusiness and ownershipType` but prior to that I assumed these were new questions being asked to the user and not just pre existing profile fields, so does it make more sense name the tests along the line of `lists existing employees, homeBasedBusiness, and ownershipType to determine fundings and certifications`, it's specific that we are expecting these exact fields to show up and the `it` block captures the behavior in it's entirety? @aloverso thoughts on this?

Reply:
done